### PR TITLE
[docs] voyager-1.7.2 changes

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-voyager/release-notes/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/migrate/live-migrate.md [hunk 1]
```diff
@@ -527,6 +526,8 @@ Create a new database user, and assign the necessary user permissions.
    SELECT 'GRANT SELECT ON ALL SEQUENCES IN SCHEMA ' || schema_name || ' TO ybvoyager;' FROM information_schema.schemata; \gexec
    ```
 
+    Note that you may get "Permission Denied" errors for `pg_catalog tables` (such as pg_statistic). These errors do not affect the migration and can be ignored.
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/live-migrate.md:529]: @aishwarya24 can we get this added to the offline migration workflow as well. In offline workflow, there is no "postgres RDS" specifically, so you can add a note in the "postgres" tab itself, at the `GRANT SELECT ON ALL TABLES IN SCHEMA` as follows:

Note that **on RDS**, you may get "Permission Denied" errors for `pg_catalog tables` (such as pg_statistic). These errors do not affect the migration and can be ignored.

File: docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md [hunk 2]
```diff
@@ -108,10 +108,14 @@ To import an updated version of the same file (that is, having the same file nam
 yb-voyager import data file --data-dir /dir/data-dir --file-table-map 'orders.csv:orders' ...
 ```
 
-After new rows are added to `orders.csv`, use the following command to load them with `--start-clean`:
+After new rows are added to `orders.csv`, use the following command to load them with the flags `--start-clean` and `--enable-upsert`.
+Note: Ensure that tables on the target YugabyteDB database do not have secondary indexes. If a table has secondary indexes, using `--enable-upsert true` may lead to corruption of the indexes.
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md:112]: Can this be made more prominent? It's essentially a warning, so want it to catch the user's attention - perhaps a box?

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md [hunk 3]
```diff
@@ -115,7 +183,7 @@ For the most accurate migration assessment, the source database must be actively
 
 1. Create a target YugabyteDB cluster as follows:
 
-    1. Create a cluster based on the sizing recommendations in the assessment report.
+    1. Create a cluster in [Enhanced Postgres Compatibility Mode](/preview/releases/ybdb-releases/v2024.1/#highlights) based on the sizing recommendations in the assessment report.
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:186]: This points to yb-db where the yugabyted instructions are present. (i.e. `--enable_pg_parity_early_access`). 
For YBA, the instructions are different - https://deploy-preview-23116--infallible-bardeen-164bc9.netlify.app/preview/releases/yba-releases/v2024.1/#highlights (you have to set GLAGs on the cluster). Can we link to both here ?

I'm not sure what to do about YBM/Aeon?

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md [hunk 4]
```diff
@@ -57,28 +57,96 @@ A sample Migration Assessment report is as follows:
...
+   CREATE USER ybvoyager IDENTIFIED BY password;
+   GRANT CONNECT TO ybvoyager;
+   GRANT <SCHEMA_NAME>_reader_role TO ybvoyager;
+   ```
+
+   If you're using [accelerated data export](#accelerate-data-export-for-mysql-and-oracle), run the additional grant as follows:
+
+   ```sql
+   GRANT FLASHBACK ANY TABLE TO ybvoyager;
+   ```
```
Comment (sanyamsinghal) [inline docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:145]: remove this part, related to accelerated data export.

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 5]
```diff
@@ -13,6 +13,29 @@ type: docs
...
+
+- Modified and enhanced the call-home diagnostic functionality. More details are collected such as, source database size, migration type (offline/live), assessment report details (unsupported features/datatypes, source index/table statistics, and so on). Furthermore, call-home diagnostic details are also sent in bulk data import and live migration workflows.
+
+- More details are being sent to [yugabyted](../../reference/configuration/yugabyted/) control plane from voyager. For example, source database size details (such as total database size, total index size, total table size, total table row count), assessment report target recommendations (total colocated size, total sharded size), analyze-schema issues, and so on.
+
+- Enhanced the cluster sizing recommendation logic in assessment report:
+
+  - Minimum recommendation for node cores is now 4.
+  - Improved recommendation logic based on experiments.
+
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/release-notes.md:34]: One more enhancement: 
- Optimised execution time of `get data-migration-report` by 90% for large databases.

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ When you run an assessment, Voyager collects metadata or metrics from the source
 The recommendations are based on testing using a [RF3](../../../architecture/docdb-replication/replication/#replication-factor) YugabyteDB cluster on instance types with 4GiB memory per core and running v2024.1.
 {{< /warning >}}
 
-Note that for cases where it is not possible to provide database access to the client machine running Voyager, you can gather the metadata from the source database using plain bash/psql scripts provided with Voyager, and then use Voyager to analyze the metadata directly.
+Note that if providing database access to the client machine running Voyager is not possible, you can gather metadata from the source database using the provided bash scripts and then use Voyager to assess the migration. Currently, these scripts require the machine to have Voyager installed due to dependencies (like psql, sqlplus clients, and so on) included with Voyager. Standalone installers are planned to be provided for these dependencies in the future.
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:36]: Let's keep this only to
`
Note that if providing database access to the client machine running Voyager is not possible, you can gather metadata from the source database using the provided bash scripts and then use Voyager to assess the migration.
`

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 7]
```diff
@@ -13,6 +13,29 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.7.2 - July 4, 2024
+
+### New features
+
+- Support for running [Assess Migration](../migrate/assess-migration/) against Oracle source database.
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/release-notes.md:20]: minor re-phrasing:  
support for Oracle Migration Assessment using assess-migration command

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md [hunk 8]
```diff
@@ -91,7 +159,7 @@ A sample Migration Assessment report is as follows:
         --source-db-schema schema1,schema2 --export-dir /path/to/export/dir
         ```
 
-    1. **Without source database connectivity**: In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts present at `/etc/yb-voyager/gather-assessment-metadata/postgresql`. You can perform the following steps with these scripts.
+    1. **Without source database connectivity**: In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts for both PostgreSQL and Oracle present at `/etc/yb-voyager/gather-assessment-metadata`. You can perform the following steps for PostgreSQL with these scripts (similar for Oracle).
 
         1. Copy the scripts to a machine which has access to the source database.
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:164]: Can you change this step to this:

On a machine which has access to the source database, copy the scripts, and install dependencies - psql, pg_dump >=14. Alternatively, you can install yb-voyager on the machine to automatically get the dependencies installed.

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md (hunk 3 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:186]: I don't think we have instructions for YBM/Aeon yet. Ill link to it as soon as we have it. But will link for YBA and YBDB

File: docs/content/preview/yugabyte-voyager/migrate/assess-migration.md [hunk 9]
```diff
@@ -161,7 +155,7 @@ A sample Migration Assessment report for PostgreSQL is as follows:
 
     1. **Without source database connectivity**: In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts for both PostgreSQL and Oracle present at `/etc/yb-voyager/gather-assessment-metadata`. You can perform the following steps for PostgreSQL with these scripts (similar for Oracle).
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/assess-migration.md:156]: For the "without source database connectivity" part, can we mention "[only PostgreSQL]"

**Review by makalaaneesh — APPROVED** [review]

**Pull request merged**